lib/scalarm/database/core/mongo_active_record_utils.rb

```diff
@@ -41,6 +41,16 @@ def parse_json_if_string(attribute)
           end
         end
 
+        def create_index(attribute)
+          @_indexed_attributes << attribute
+          # puts "Create index: #{attribute} - #{@_indexed_attributes}"
+        end
+
+        def inherited(subclass)
```

Comment:
@dkrol - OK, u mnie w RubyMine nie pokazała się ikonka override. Czy nie warto byłoby dodać do dokumentacji klasy (komentarz nad nazwą klasy) informację, że przy dziedziczeniu dzieją się rzeczy niestandardowe? Motywacją jest to, że osoba używająca kiedyś tej klasy musiałaby przeglądnąć porządnie cały kod, żeby zobaczyć takie rzeczy. Przy okazji w dokumentacji ``MongoActiveRecord`` można napisać jak działa u nas "system indeksów".